### PR TITLE
Move to codesandbox org

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
**What kind of change does this PR introduce?**
Changing all @CompuIves' GitHub references to the new @codesandbox organization

**What is the current behavior?**
Links go to @CompuIves' repos

**What is the new behavior?**
Links go to @codesandbox's repos

**Checklist**:

- [ ] Documentation N/A
- [ ] Tests N/A
- [x] Ready to be merged 
- [x] Added myself to contributors table

### CODE REVIEW

File: packages/app/integration-tests/tests/sandboxes.test.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const SANDBOXES = [
   'o29j95wx9',
   'k3q1zjjml5',
   'github/reduxjs/redux/tree/master/examples/real-world',
-  'github/CompuIves/codesandbox-presentation',
+  'github/codesandbox/codesandbox-presentation',
```
Comment (CompuIves) [inline packages/app/integration-tests/tests/sandboxes.test.js]:
```suggestion
  'github/CompuIves/codesandbox-presentation',
```

This one needs to stay the same, that one is still on my personal profile 😄

Comment (MichaelDeBoey) [reply packages/app/integration-tests/tests/sandboxes.test.js]: Oops 😓

**Comment by MichaelDeBoey** [comment]
Resolved discussions 🙂

**Comment by CompuIves** [comment]
Great! Thanks for the contribution!!

**Comment by MichaelDeBoey** [comment]
With pleasure 🙂

**Pull request merged**